app/javascript/admin/controllers/config_controller.js

```diff
@@ -157,6 +157,8 @@ export default class ConfigController extends Controller {
     event.preventDefault();
     this.configModalAnchorTarget.innerHTML = adminModal({
       title: emailAuthModalTitle,
+      controllerName: 'config',
```

Comment:
The changes in this particular file are necessary based on the slight tweak I made to the `adminModal.js` file.